styles.css

```diff
@@ -255,9 +264,101 @@ main {
   color: white;
 }
 
+.popup-window {
+  background-color: pink;
+  z-index: 100;
+  opacity: 0.9;
```

Comment:
Kindly adjust the margin between the card the the blurred background to match the design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/232060778-1750246a-83e1-45d9-89d5-e7bf0cd6e9ad.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/232060975-57fd6234-6f16-4a84-ad81-a6426a2213aa.png)